web/app/js/components/MetricsTable.jsx

```diff
@@ -128,7 +128,9 @@ const columnDefinitions = (resource, showNamespaceColumn, PrefixedLink, isTcpTab
       let nameContents;
       if (resource === "namespace") {
         nameContents = <PrefixedLink to={"/namespaces/" + d.name}>{d.name}</PrefixedLink>;
-      } else if (!d.added || isAuthorityTable) {
+      } else if (isAuthorityTable) {
+        nameContents = getResourceDisplayName(d).split('.')[0];
```

Comment:
What happens for non-service authorities?  For example, when Prometheus calls directly to a pod ip, would this display the first octect of the IP address?  e.g. if the authority is `5.6.7.8:99` then would this display `5` as the service name?  

One idea would be to only reformat names with hosts that end in `svc.cluster.local` and keep all other names as-is.

Reply:
Or omit non-`svc.cluster.local` entries entirely.